pandas/tseries/tests/test_resample.py

```diff
@@ -2043,17 +2044,19 @@ def test_resample_basic(self):
         assert_series_equal(result2, expected)
 
     def test_resample_empty(self):
-
         # GH12771 & GH12868
         index = PeriodIndex(start='2000', periods=0, freq='D', name='idx')
         s = Series(index=index)
 
-        expected_index = PeriodIndex([], name='idx', freq='M')
-        expected = Series(index=expected_index)
+        for freq in ['M', 'D', 'H']:
+            expected_index = PeriodIndex([], name='idx', freq=freq)
+            expected = Series(index=expected_index)
 
-        for method in resample_methods:
-            result = getattr(s.resample('M'), method)()
-            assert_series_equal(result, expected)
+            for method in resample_methods:
+                result = getattr(s.resample(freq), method)()
+                if method != 'ohlc':  # returns df
+                    assert_series_equal(result, expected, check_dtype=False)
+                self.assertEqual(result.index.freq, freq)
```

Comment:
`ohlc` I agree, that needs a direct comparison

count / size are already tested - that a Series has a PeriodIndex (or an empty PeriodIndex!) can't affect the dtypes of its Series.

I could check the indexes are the same though, that's worthwhile